test/index.js

```diff
@@ -0,0 +1,15 @@
+import assert      from 'assert'
+import startServer from '../lib/server'
+
+const server = startServer()
+
+describe('/version', () => {
+  it('returns the current version', (done) => {
+    server.inject({
+      url: '/version'
+    }, (res) => {
```

Comment:
If you wanted, you could even reduce this down to `server.inject("/version", ....)`